requirements.txt

```diff
@@ -0,0 +1,47 @@
+appnope==0.1.0
```

Comment:
Are this and the two new dev-requirements files needed for something?

Reply:
Hmm nope - I'm guessing these files were probably moved and my branch still had the old files here. Merging with master was quite an ordeal since it was such an old PR - seems like I still made some mistakes merging here and there! I'll clean this up.

Reply:
OK great! yes, we will do our best avoid such long-festering PRs in the future!

Reply:
Yeah no worries! :)